### ISSUE TITLE
Edit forms should be keyboard accessible

### ISSUE THREAD

**Issue by patrickweaver** [patrickweaver/website-editor#1]
Esc key should be the same as "Cancel" button, Enter should be the same as "Update" button with Alt-Enter for new lines?
